.github/actions/javascript/getPullRequestDetails/getPullRequestDetails.js

```diff
@@ -87,9 +87,13 @@ if (pullRequestNumber) {
         ...DEFAULT_PAYLOAD,
         state: 'all',
     })
+        .then(({data}) => _.find(data, PR => PR.user.login === user && titleRegex.test(PR.title)).number)
```

Comment:
Do we care about highlighting the case where we fail to get a matching PR? I think probably not as the logs should be clear enough (like the one we're currently fixing), but thought I'd ask.